README.md

```diff
@@ -426,10 +482,40 @@ var nullsSkipped = qs.stringify({ a: 'b', c: null}, { skipNulls: true });
 assert.equal(nullsSkipped, 'a=b');
 ```
 
+If you're communicating with legacy systems, you can switch to `iso-8859-1`
+using the `charset` option:
+
+```javascript
+var iso = qs.stringify({ æ: 'æ' }, { charset: 'iso-8859-1' });
+assert.equal(iso, '%E6=%E6');
+```
+
+Characters that don't exist in `iso-8859-1` will be converted to numeric
+entities, similar to what browsers do:
+
+```javascript
+var numeric = qs.stringify({ a: '☺' }, { charset: 'iso-8859-1' });
+assert.equal(numeric, 'a=%26%239786%3B');
+```
+
+You can use the `utf8Sentinel` option to announce the character by
+including an `utf8=✓` parameter with the proper encoding if the checkmark,
+similar to what Ruby on Rails and others do when submitting forms.
+
+```javascript
+var sentinel = qs.stringify({ a: '☺' }, { utf8Sentinel: true });
+assert.equal(sentinel, 'utf8=%E2%9C%93&a=%E2%98%BA');
+
+var isoSentinel = qs.stringify({ a: 'æ' }, { utf8Sentinel: true, charset: 'iso-8859-1' });
```

Comment:
That's for the case I described in https://github.com/ljharb/qs/pull/268#discussion_r206413387

When you have to assume that the encoding is `iso-8859-1` unless there's a sentinel that says otherwise.

Reply:
(Sorry about my earlier comment, I thought this was in `qs.parse` :)

It might be a slim use case, but including a sentinel that explicitly says that the query string is in `iso-8859-1` could be useful in some scenarios. Obviously it's much more useful in the inverse case, because all of this has to do with legacy services that just assume it's `iso-8859-1`.

Reply:
Only because the parameter is called `utf8`. I agree that the name op the option isn’t ideal, but couldn’t come up with anything else. I’ve tried googling it to see if there’s an established name for the concept, but nothing turned up.

Reply:
charsetSentinel? encodingSentinel?

When you say "could be useful", is this a use case you've ever seen anywhere? Or are we just planning for something that *might* happen?

Reply:
I like `charsetSentinel` better, let's go with that instead, fixed in 2efe555b8bef0e03568ac4a465ff97304c15bfef.

Well, the test suite for that `FormMail.pl`-compatible service could have used the ability to encode a query string and an `application/x-www-form-urlencoded` body with an iso-8859-1 sentinel, as that's one of the cases that explicitly had to be supported.

In other words, it's useful whenever you want to emulate what a browser sends when a form containing `utf8=✓` is submitted. So mostly in test suites for web servers that cannot assume that everything is utf-8.